doc/nrf/ug_zigbee_configuring.rst

```diff
@@ -95,8 +95,7 @@ You can enable the following additional configuration options:
     For example, you can set channels 13, 16, and 21.
     You must have at least one channel enabled with this option.
 
-* :option:`CONFIG_IEEE802154_VENDOR_OUI_ENABLE` - MAC Address Block Large is set to Nordic Semiconductor's MA-L block (f4-ce-36) by default.
-  To set a different MA-L, enable this option and edit the :option:`CONFIG_IEEE802154_VENDOR_OUI` to the desired value.
+.. include:: /includes/ieee802154_eui64_conf.txt
 
```

Comment:
I think it's a good place to describe how to verify EUI-64 address in case of Zigbee:

```suggestion

#. Check the EUI-64 value using :ref:`lib_zigbee_shell`:

    .. code-block:: console

      > zdo eui64
      8877665544332211
      Done
```